test/vector_test.rb

```diff
@@ -307,6 +307,43 @@ def test_each_with_index
 
   # converters
 
+  def test_to_integer
+    vector = Rover::Vector.new([1.0,2.0,3.0, nil])
+    vector = vector.to(:integer)
+    assert_equal vector[0], 1
+    assert_equal vector.to_numo.class, Numo::Int64
+
+    vector = Rover::Vector.new(["1","2","3"])
+    vector = vector.to(:integer)
+    assert_equal vector[0], 1
+    assert_equal vector[1], 2
+    assert_equal vector[2], 3
+    assert_equal vector.to_numo.class, Numo::Int64
+  end
+
+  def test_to_float
+    vector = Rover::Vector.new([1,2,3, nil])
+    vector = vector.to(:float)
+    assert_equal vector[0], 1.0
+    assert_equal vector.to_numo.class, Numo::DFloat
+
+    vector = Rover::Vector.new(["1.0","2.1",nil])
+    vector = vector.to(:float)
+    assert_equal vector[0], 1.0
+    assert_equal vector[1], 2.1
+    assert_equal vector[2], nil
+    assert_equal vector.to_numo.class, Numo::DFloat
+  end
+
+  def test_to_boolean
+    vector = Rover::Vector.new([1,2,0])
+    vector = vector.to(:boolean)
+    assert_equal vector[0], 1
```

Comment:
We should fix the current behavior to have this return `true` / `false`.

Reply:
Makes sense. I might leave that for another PR though.